CMakeLists.txt

```diff
@@ -44,9 +44,11 @@ if(MSVC)
 else()
     option(BUILD_SHARED_LIBS "Build libraries as shared as opposed to static" ON)
 endif()
-if (NOT BUILD_SHARED_LIBS)
-    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-endif()
+
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
What is the rationale for this change?

Reply:
> What is the rationale for this change?

So this was a trap?!

Reply:
> So this was a trap?!

I simply forgot about what our recommended  way of handling `CMAKE_POSITION_INDEPENDENT_CODE` was, and I had to check `how-to-export-cpp-library` to remember the details.